bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBatch.java

```diff
@@ -366,4 +375,12 @@ public void testBatchGets() throws Exception {
 
     table.close();
   }
+
+  public void testBatchDoesntHang() throws Exception {
+    Connection closedConnection = ConnectionFactory.createConnection(IntegrationTests.getConfiguration());
+    Table table = closedConnection.getTable(TABLE_NAME);
+    closedConnection.close();
+    thrown.expect(RetriesExhaustedWithDetailsException.class);
```

Comment:
If the specifics of the exception on .close() change, it should not require changes of Batch testing, which is what we're testing here.  Getting into the specific details of the .close() exception here would require future changes.  I'd like to avoid those future changes and test specific behavior in unit tests, if that's ok.